kong/constants.lua

```diff
@@ -202,6 +202,25 @@ local constants = {
   CLUSTERING_OCSP_TIMEOUT = 5000, -- 5 seconds
 
   CLEAR_HEALTH_STATUS_DELAY = 300, -- 300 seconds
+
+  LOG_LEVELS = {
+    debug = ngx.DEBUG,
+    info = ngx.INFO,
+    notice = ngx.NOTICE,
+    warn = ngx.WARN,
+    error = ngx.ERR,
+    crit = ngx.CRIT,
+    alert = ngx.ALERT,
+    emerg = ngx.EMERG,
+    [ngx.DEBUG] = "debug",
```

Comment:
should this be 
```lua
debug = ngx.DEBUG,
[ngx.DEBUG] = ngx.DEBUG?
```
otherwise if user set log level to `"debug"`, it won't be same as `ngx.DEBUG` (which is `8`). the end result is not harmful, but it breaks the logic we did above that "don't call the API again
if current level is same as level to be set".

Reply:
if user sets log level to `debug`, it will be `ngx.DEBUG` (8)

if current level is the same as the level to be set, the API will return:

```
200 OK
{
    "message": "log level is already notice"
}
```